Resources/Prototypes/_StarLight/Entities/Structures/Machines/Computers/computers.yml

```diff
@@ -0,0 +1,52 @@
+#abductor
+- type: entity
+  parent: BaseComputer
+  id: SyndicateObservationConsole
+  name: syndicate observation console
+  description: A computer that is used for spying on the station.
+  components:
+  - type: Sprite
+    sprite: _Starlight/Structures/Machines/abductor_camera_console.rsi
+    layers:
+    - map: ["console"]
+      state: console
+    - map: ["computerLayerScreen"]
+      state: comm_syndie
+  - type: AbductorHumanObservationConsole
+  - type: ActivatableUI
+    key: enum.AbductorCameraConsoleUIKey.Key
+  - type: UserInterface
+    interfaces:
+     enum.AbductorCameraConsoleUIKey.Key:
+        type: AbductorCameraConsoleBui
+  - type: PointLight
+    enabled: true
+    radius: 1.5
+    energy: 1.6
+    color: "#FC0097"
+
+- type: entity
+  parent:
+  - Incorporeal
+  - BaseMob
+  id: AbductorHumanObservationConsoleEye
+  name: abductor eye
+  description: The abductor's viewer.
+  categories: [ HideSpawnMenu ]
+  suffix: DO NOT MAP
+  components:
+  - type: NoFTL
+  - type: WarpPoint
```

Comment:
should this be a warp point? if not for this, then i'd say this should probably just parent the ai eye rather than copy-pasting its values